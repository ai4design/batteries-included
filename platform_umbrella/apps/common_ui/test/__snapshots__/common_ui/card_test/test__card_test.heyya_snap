<div class="p-4 bg-white rounded-lg border border-gray-200 shadow-lg flex flex-col">
  
  
  Inner test

</div>