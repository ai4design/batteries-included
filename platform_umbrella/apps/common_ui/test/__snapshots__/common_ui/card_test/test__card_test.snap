"<div class=\"p-4 bg-white rounded-lg border border-gray-200 shadow-lg flex flex-col\">\n  \n  \n  Inner test\n\n</div>"